yocto/meta-sabaton/recipes-connectivity/cyclonedds/cyclonedds_0.10.1.bb

```diff
@@ -0,0 +1,7 @@
+include cyclonedds.inc
+LICENSE = "Eclipse-2"
+SRC_URI = "git://github.com/eclipse-cyclonedds/cyclonedds.git;protocol=https;branch=releases/0.10.x;tag=0.10.1"
```

Comment:
Latest cyclone release is https://github.com/eclipse-cyclonedds/cyclonedds/releases/tag/0.10.2, not 0.10.1.

File should be renamed to cyclonedds_0.10.2.bb and point to the 0.10.2 release.